src/dynarec/arm64/dynarec_arm64_660f.c

```diff
@@ -1084,7 +1084,22 @@ uintptr_t dynarec64_660F(dynarec_arm_t* dyn, uintptr_t addr, uintptr_t ip, int n
                         SMWRITE2();
                     }
                     break;
-
+                case 0x17:
+                    INST_NAME("EXTRACTPS Ew, Gx, Ib");
+                    nextop = F8;
+                    GETGX(q0, 0);
+                    if (MODREG) {
+                        ed = xRAX+(nextop&7)+(rex.b<<3);
+                        u8 = F8&0b11;
+                        MOVx_REG(ed, xZR);
+                        VMOVSto(ed, q0, u8);
+                    } else {
+                        addr = geted(dyn, addr, ninst, nextop, &wback, x2, &fixedaddress, NULL, 0xfff, 0, rex, NULL, 0, 0);

```

Comment:
geted as a delta of `1` here, because the opcode have an extra byte attached

Reply:
it should be `0xfff<<2` and a mask of `3` here, because it's a `STW` opcoration, wich is 32bits write.